src/NextGen/WelcomeBanner/resources/views/welcome-banner.php

```diff
@@ -75,31 +86,35 @@
                     margin: 0;
                     font-size: 18px;
                     color: var(--givewp-blue-500);
-                "><?php _e('Next Steps', 'give'); ?></h2>
+                "><?php
+                    _e('Next Steps', 'give'); ?></h2>
 
-                <div style="display:flex;flex-direction:column;gap:73px;">
+                <div
+                    style="height: calc(100% - 26px); display:flex; flex-direction:column; justify-content: space-between; gap:26px;">
 
                     <!-- Join the Journey -->
-                    <div>
-                        <h3 style="font-size: 16px;margin: 12px 0;"><?php _e('Join the Journey', 'give'); ?></h3>
-                        <p style="font-size: 12px;"><?php _e('Stay informed about the Visual Donation Form Builder beta as well as GiveWP version 3.0', 'give'); ?></p>
-                        <givewp-welcome-banner-newsletter></givewp-welcome-banner-newsletter>
-                        <div id="givewpNextGenNewsletterSuccess" style="
-                            visibility: hidden;
-                            margin-top: var(--givewp-spacing-4);
-                        ">
-                            <div style="flex:1;">
-                                <?php _e('Awesome, you’re in! You will be among the first to receive updates.', 'give'); ?>
-                            </div>
-                        </div>
+                    <div style="display: flex; flex-direction: column;">
+                        <h3 style="font-size: 16px; margin: 12px 0;"><?php
+                            _e('Participate in a Beta Focus Group And Get a $25 Amazon Gift Card', 'give'); ?></h3>
+                        <p style="font-size: 12px; margin: 0 0 12px 0;"><?php
+                            _e('Join us on <strong>11 May 2023</strong> to provide focused feedback about our new visual donation form builder. Spacing is limited to <strong>eight(8)</strong> participants per session.',
+                                'give'); ?></p>
+                        <a href="https://givewp.com/focus-groups/" target="blank"
+                           style="flex: 1; font-size: 14px; text-align: center; color: white; padding: 12px 24px;border-radius: 2px; border: solid 1px var(--givewp-green-500); background-color: var(--givewp-green-500); text-decoration: none;">
+                            <?php
+                            _e('Three(3) time slots available - Register now', 'give'); ?>
```

Comment:
```suggestion
                            _e('Three (3) time slots available - Register now', 'give'); ?>
```

Should there be a space added here?

Reply:
I can add it. Resolved: https://github.com/impress-org/givewp-next-gen/pull/176/commits/cb9168929edbe9a762cdba20fc3122ee3c3bfabe. Should we add the space between it for "eight(8)" as well?

Reply:
Yes, I would add the space in that situation.

Reply:
Resolved: https://github.com/impress-org/givewp-next-gen/commit/2ca83b87687da97930d61d599e5a79a9cfa8c5a6